docs/tutorial.md

```diff
@@ -148,8 +184,7 @@ $ jj log
 ```
 
 The `@` indicates the working-copy commit. The first ID on a line
-(e.g. "mpqrykyp" above) is the "change ID", which is an ID that follows the
-commit as it's rewritten (similar to Gerrit's Change-Id). The second ID is the
+(e.g. "mpqrykyp" above) is the "change ID". The second ID is the
```

Comment:
Drop the quotes around "change ID" now that we've introduced the term earlier. Can also drop the "which changes when you rewrite the commit" on the next line, i think

Reply:
Done in https://github.com/martinvonz/jj/commit/ebd4f999b45a2d22154887a0f80ff25e3a92f042